test/nodetool/test_describecluster.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+from rest_api_mock import expected_request
+
+
+def test_describering(nodetool):
+    cluster_name = "Test Cluster"
+    snitch_name = "org.apache.cassandra.locator.SimpleSnitch"
+    partitioner = "org.apache.cassandra.dht.Murmur3Partitioner"
+    dynamic_snitch = "disabled"
+    schema_versions = [{"key": "83541f18-c1bc-11ee-b55e-4d563ca8da4b",
```

Comment:
Lets have a test where we have multiple schema versions too.
It can be a test parameter, using @pytest.parametrize().

Reply:
done